test/pageobjects/admin.login.page.ts

```diff
@@ -0,0 +1,32 @@
+import { $ } from "@wdio/globals";
+import Page from "./page.js";
+
+class AdminLoginPage extends Page {
+  public get emailField() {
+    return $('input[name="username"]');
+  }
+  public get adminLoginPageHeader() {
+    return $('div[id="header"]');
+  }
+  public get passwordField() {
+    return $('input[name="password"]');
+  }
+  public get logInButton() {
+    return $('input[value="Log in"]');
+  }
+
+  public open() {
+    return browser.url(`https://dev.rentzila.com.ua/api/admin/`);
```

Comment:
@Anastas11aZakharova Btw, you already have the 'open' method in the 'page' class, use it here. I mean, you can set the parameter to the 'open' method mentioned and modify the link, dependly on the page. Keep in mind, that the methods from the 'page' class must be usable for each 'page-object' file, no need for it in other case.

Reply:
Fixed